<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersGMMClust</class>
 <widget class="QWidget" name="ParametersGMMClust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Initialization</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="gmmCount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Number of mixtures trained per class</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="gmmInitCombo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Method for initialization of the GMM prior to the first EM step
Random: randomly place the means of the mixtures (unit variance)
Uniform: uniformly split the space along the first axis and set the means there (unit variance)
K-Means: perform K-Means and assign means and variance to each mixture</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Random</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Uniform</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>K-Means</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>48</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Covariance Matrix</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mixture Components</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gmmCovarianceCombo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Covariance Matrix type</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Full</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Diagonal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spherical</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="marginalsButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marginals</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
